src/core/services/theming/theming.js

```diff
@@ -585,6 +585,9 @@ function ThemingProvider($mdColorPalette, $$mdMetaProvider) {
    *
    *       $mdTheming.defineTheme('myTheme', {
    *         primary: 'blue',
+   *         primaryHues: {
```

Comment:
I'm not sure if this doc makes things clear. The `setPalette` function seems to have no docs itself.
Any suggestions?

Reply:
Thank you very much for this thorough information. I really appreciate your feedback.
I tried to implement it as good as possible - though I'm not the best at writing docs.

Sadly, I'm running out of time for this. So if you should have other suggestions regarding documentation I'd like to ask you to make these changes directly, if that is possible.

Reply:
Unfortunately, it's non-trivial for me to make changes to your PR, but I do think that this is about ready to wrap up. Thank you for your cooperation.